### ISSUE TITLE
DeDRM Windows Application v6.62?

### ISSUE THREAD

**Issue by Footprintxabc** [apprenticeharper/DeDRM_tools#784]
Does anyone know if the DeDRM Windows Application have been fixed in v6.62? It's been broken since 6.5.5, and would like to ask before trying. The update notes didn't mention anything on that, so would be nice to know. Much appreciated.

**Comment by ElleKayEm** [comment]
I've seen a few issues here where people get 6.6.1 working by installing pylzma.  Same thing would probably work for 6.6.2.

**Comment by Footprintxabc** [comment]
> I've seen a few issues here where people get 6.6.1 working by installing pylzma. Same thing would probably work for 6.6.2.

Thank you for letting me know. I'll still be waiting on the side for an official fix in the future releases.

**Comment by ElleKayEm** [comment]
Not sure there's a fix beyond updating the instructions to say that pylzma should be installed in addition to Python and PyCrypto.

Also, have you already seen this?

https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md#the-windows-application

**Comment by Intervencion** [comment]
6.6.2 here.
Installed ActiveState Python, PyCrypto and pylzma (all for ActivePython 2.7.14.2717) Also moved the TCL's folder files to the Lib folder.

When I drag an azw3 to the bat it does nothing

`C:\DeDRM_tools\DeDRM_Windows_Application\DeDRM_App>chcp 65001   1>nul  && set PWD="C:\DeDRM_tools\DeDRM_Windows_Application\DeDRM_App\"   && cd /d "C:\DeDRM_tools\DeDRM_Windows_Application\DeDRM_App\DeDRM_lib"   && start /min python2.7 DeDRM_App.pyw C:\DeDRM_tools\DeDRM_Windows_Application\DeDRM_App\kino_no_Tabi_the_Beautiful_Wor_-_Shigure_Sawa_Keiichi.azw3

C:\DeDRM_tools\DeDRM_Windows_Application\DeDRM_App\DeDRM_lib>PAUSE
`

Let me know if I can provide further info to help debug this, thanks.

**Comment by ElleKayEm** [comment]
Have you been able to double-click it and add your E-ink Kindle serial number?

**Comment by Intervencion** [comment]
It just opens and closes (Hence the PAUSE I added to the bat).

**Comment by ElleKayEm** [comment]
A couple things... the python scripts for the plugin and the stand-alone are the same.  The difference is the plugin is using calibre's built in Python.  Also, are you using calibre's ebook viewer after removing DRM with the plugin?  Because that might be the problem -- not the ebook itself.

**Comment by Intervencion** [comment]
Isn't it supposed to output a new eBook (drmless) and rename my original azw3 (with drm) to like .azw3.bk?

**Comment by ElleKayEm** [comment]
I believe the stand alone adds _nodrm to the new file name.  So "filename_nodrm.azw3".

You said you were only using the stand-alone to see if it didn't mess up the text and character orientation.  If you could get the stand-alone working, the results might be the same because it's the same scripts, just run with installed Python vs calibre's built-in Python.  But you might try viewing the DRM-free book with something besides calibre's ebook viewer.  It opens lots of book types, but doesn't always display things that well.

**Comment by Intervencion** [comment]
Oh rip, nothing then. If it's the same output than Calibre's then is not good for what I want.

Thank you!

**Comment by ElleKayEm** [comment]
The results might be different, but I'd be surprised.

Again if you've only looked at it with calibre's ebook viewer, then it might not be displayed correctly.  Try opening it with Kindle for PC.

**Comment by Intervencion** [comment]
They look as they should on Kindle. Is just Calibre refusing to fix years old bugs.

**Comment by ElleKayEm** [comment]
Well, calibre is frequently updated, but the ebook viewer might be lower priority.  And it's made to view practically all book formats, but it may not do specific attributes for each that well.

**Comment by Intervencion** [comment]
No, the dev on the bug thread said they had no plans to fix it, back in 2015. Anyways we're offtopic now X) 
Thanks for the help regardless, and thank you apprenticeharper for the tools.
